app/controllers/users_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class UsersController < ApplicationController
   def index
-    @users = User.order(:id).page(params[:page])
+    @users = User.order(:id).page(params[:page]).includes(image_attachment: :blob)
```

Comment:
`has_one_attached :image` で `with_attached_image` という scope が使えるようになるのでそちらでも OK です 💁 
https://blog.saeloun.com/2020/03/06/eagerload-active-storage-models.html#with_attached_name

```suggestion
    @users = User.with_attached_image.order(:id).page(params[:page])
```